chart/templates/server-deployment.yaml

```diff
@@ -40,9 +40,33 @@ spec:
       securityContext:
         fsGroup: {{ $root.Values.fsGroup }}
       volumes:
+        - name: vets-api-mockdata
```

Comment:
Good to know - it will be easier for you to make that change when this is pushed in - at this point we want to stay as close to like for like as possible. We would prefer to not have breakage because we tried to centralize the config and change the configs at the same time.

Reply:
Would it make sense to add `{{- if $root.Values.mockData.enabled }}` to this volume?

Reply:
in this particular case, no, since the concern is over inclusions of volumes in this context that should never be in this context. There are here now because of the $root.Values.common.volumes  list in values.yaml which contained all three of these volumes. I iwould like to initial leave them here so that when we first deploy  we will remain consistent with the settings that are current in the static environments. After we have successful rollout of these changes we can double back and remove the un-needed volumnes. I would prefer to keep the targeted resource setup changes to a minimum as we make the helm chart structural changes.